apps/site/util/getNodeDownloadSnippet.ts

```diff
@@ -5,11 +5,23 @@ import type { NodeRelease } from '@/types';
 import type { PackageManager } from '@/types/release';
 import type { UserOS } from '@/types/userOS';
 
-export const getNodeDownloadSnippet = (
+const latestVersion = fetch('https://latest.nvm.sh', { redirect: 'manual' })
```

Comment:
That's intentional, so that every invocation of the function shares the same version value. The `await` would reduce concurrency and make things slower.